build-tools/validate-e2e

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
A pretty much identical file exists at build-tools/github/validate-e2e. The only difference is that this file has less debug output.